static/downloadables/docs/index.rst

```diff
@@ -24,7 +24,7 @@ To download the PRSKB CLI tool, head over to the `PRSKB website download page <h
 
 Given the required parameters, the tool will calculate risk scores for each individual sample for each study and trait combination in the database. 
 
-*NOTE: You MUST have bash and jq for bash, and python3 and the PyVCF and filelock Python modules installed.*
+*NOTE: You MUST have bash and jq for bash, and python3 and the PyVCF, filelock, and requests Python modules installed.*
```

Comment:
In the future, when there are changes to be made to this file, and then the other files that are created using this file, just make the changes here and then let me know so I can update them properly. (If you want more information about this type of file, look into readthedocs. Here is a link to get started: https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html)